code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -23,7 +23,14 @@ meteor_act
 		else
 			P.on_hit(src, def_zone)
 			return 2
-	//Checking abosrb for spawning shrapnel
+	//Checking to see if we're wearing a spacesuit and whether it's been punctured.if not, we cannot embed shrapnel... usually. it still happens very rarely when blunt attacks deal high enough damage.
+	for(var/obj/item/clothing/suit/space/SS in src.get_equipped_items())
```

Comment:
Same as above, this should not be handled here at all.

Reply:
All the requested changes have been made. Have a good one mate